templates/_components/quote_and_image.html.twig

```diff
@@ -0,0 +1,36 @@
+<div class="govuk-grid-row govuk-!-padding-9">
+    <div class="govuk-grid-column-one-quarter govuk-heading-xl govuk-!-padding-left-5" style="color: var(--sub-color); text-align: right; width: 5%;">“</div>

```

Comment:
Inline styles should not be used unless absolutely necessary. It often creates maintenance problems further down the line.

These styles should be moved into an appropriate SCSS (css) file.

If you want to target only this specific element, then I would recommend adding a new class to it which you can use to select it in the CSS.